doc/cephfs/experimental-features.rst

```diff
@@ -44,12 +44,10 @@ some cases. Specifically, if you share a pool for multiple FSes and delete
 a snapshot in one FS, expect to lose snapshotted file data in any other FS using
 snapshots. See the :doc:`/dev/cephfs-snapshots` page for more information.
 
-Snapshots are known not to work with multi-MDS filesystems.
-
 For somewhat obscure implementation reasons, the kernel client only supports up
 to 400 snapshots (http://tracker.ceph.com/issues/21420).
 
-Snapshotting was blocked off with the "allow_new_snaps" flag prior to Firefly.
+Snapshotting was blocked off with the ``allow_new_snaps`` flag prior to Firefly.
```

Comment:
snapshot is enabled by default. I think this line should be
Snapshotting was blocked off with the ``allow_new_snaps`` flag prior to Mimic.

Reply:
@ukernel Done.